app/views/customers/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1><%=@customer.name%>'s Items</h1>
+<%@customer.items.each do |item|%>
+    <div id="item-<%= item.id %>">
+        <h3><%=item.name%></h3>
+        <ul>
+            <li>Price: $<%=item.price%></li>
+            <li>Supermarket: <%=item.supermarket_name%></li>
+        </ul>
+    </div>
+<%end%>
+
+<%= form_with(url: "/customers/#{@customer.id}/items", method: :post ) do |f| %>
```

Comment:
Awesome work setting up this form! This path looks great, and is very RESTful. Another option could be `"/customer_items"` but I usually prefer the path you used because you can easily add the customer.id in the path rather than sending it as a hidden field. Both are RESTful though